examples/remix/package.json

```diff
@@ -3,24 +3,28 @@
   "sideEffects": false,
   "scripts": {
     "build": "remix build",
-    "dev": "remix dev"
+    "dev": "remix dev",
```

Comment:
Should we use the new dev server?

Reply:
@leerob As you can see in https://github.com/remix-run/remix/pull/6650, the only thing that changed for `v2_dev` is enabling the future flag